joomla/joomla-cms
# Add useragent for update calls

Pull Request for Issue #15381

### Summary of Changes

Add useragent for update calls

### Testing Instructions

##### test 1
- install a outdated extension
- apply this patch
- run the update
- confirm everything works

##### test 2

- install a outdated joomla version
- patch the file manually
- update to the last joomla version

### Expected result

Update works with the useragent checks

### Actual result

Update works but have no useragent

### Documentation Changes Required

None
